### PR TITLE
Update File.xml

### LINKED ISSUE #2525: Missing possible exceptions in System.IO.File documentation
The documentation of method members "Copy", "Create", "CreateText", "WriteAllBytes", "WriteAllLines", "WriteAllText"  does not mention that "UnauthorizedAccessException" will be thrown if any of those methods try to create/override existing file with the "Hidden" attribute.

### PR DESCRIPTION
**Pull request by kaloyankrastev**
## Summary

Include missing possible exceptions that might be thrown in some System.IO.File methods.

Fixes #2525

### CODE REVIEW

File: xml/System.IO/File.xml [hunk 1]
```diff
@@ -848,7 +848,11 @@
   
  -or-  
   
- <paramref name="destFileName" /> is read-only.</exception>
+ <paramref name="destFileName" /> is read-only.
+        
+ -or-  
+  
+ <paramref name="destFileName" /> exists and is hidden, but <paramref name="sourceFileName" /> is not.</exception>
```
Comment (rpetrusha) [inline xml/System.IO/File.xml]:
```suggestion
<paramref name="overwrite" /> is <see langword="true" />, <paramref name="destFileName" /> exists and is hidden, but <paramref name="sourceFileName" /> is not hidden.</exception>
```

**Comment by kaloyankrastev** [comment]
Hi @rpetrusha,
thanks for having time looking at my pull request. Your suggestion makes perfect sense and I directly committed it, but there is back end engine exception in one of the build tasks. I can not find a proper way to retry the checks. Should I make a a commit with a dummy change in order to trigger the checks?

Again, thanks for taking my suggestions into account!

**Comment by MSDN-WhiteKnight** [comment]
Try closing/reopening pull request, it should trigger build.

**Comment by rpetrusha** [comment]
Thanks for making the additional change, @kaloyankrastev. Your closing and reopening your PR did trigger a new build, which looks good. I'll approve and merge now. The changes should be live on docs.microsoft.com in the next day or two.

**Pull request merged**